From 570e064981924b00f6b8afd80f167e134aa6f051 Mon Sep 17 00:00:00 2001
From: Andrey Borzenkov <arvidjaar@gmail.com>
Date: Wed, 24 Jun 2015 09:52:15 +0100
Subject: [PATCH 904/907] chkconfig: Do not show errors in service lists.

(cg) This is a modified version of original patch but is logicially identical.

I'm not actually sure we want this.
---
 chkconfig.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/chkconfig.c b/chkconfig.c
index e33302c..f70db48 100644
--- a/chkconfig.c
+++ b/chkconfig.c
@@ -427,12 +427,6 @@ static int showServiceInfo(struct service s, int forgiving) {
 	    }
     }
 
-    if (rc) {
-	if (!forgiving)
-	    readServiceError(rc, s.name);
-	return forgiving ? 0 : 1;
-    }
-
     printf("%-15s", s.name);
     if (s.type == TYPE_XINETD) {
 	    printf("\t%s\n", s.levels ? _("on") : _("off"));
-- 
2.3.2

